lib/utils/index.ts

```diff
@@ -1,25 +1,30 @@
-import { isExportStory } from '@storybook/csf'
-import { ExportDefaultDeclaration } from '@typescript-eslint/types/dist/ast-spec'
+import { IncludeExcludeOptions, isExportStory } from '@storybook/csf'
+import {
+  ExportDefaultDeclaration,
+  ExportNamedDeclaration,
+  Identifier,
+  ObjectExpression,
+} from '@typescript-eslint/types/dist/ast-spec'
 import { ASTUtils } from '@typescript-eslint/experimental-utils'
 import {
   isFunctionDeclaration,
   isIdentifier,
   isObjectExpression,
+  isSpreadElement,
   isTSAsExpression,
   isVariableDeclaration,
   isVariableDeclarator,
 } from './ast'
+import { RuleContext } from '@typescript-eslint/experimental-utils/dist/ts-eslint'
 
-export const docsUrl = (ruleName: any) =>
+export const docsUrl = (ruleName: string) =>
   `https://github.com/storybookjs/eslint-plugin-storybook/blob/main/docs/rules/${ruleName}.md`
 
-export const isPlayFunction = (node: any) => {
```

Comment:
why was this deleted?

Reply:
As far as I remember, it was not used at all.

Reply:
That makes sense. I thought I saw it being used somewhere. Thanks!